thesheriff/domain/outlaw/repository/outlaw_repository.py

```diff
@@ -13,7 +13,7 @@ def of_id(self, outlaw_id: int) -> Outlaw:
         pass
 
     @abc.abstractmethod
-    def add(self, new_outlaw: Outlaw) -> int:
+    def add(self, new_outlaw: Outlaw) -> Outlaw:
```

Comment:
I prefer to return the entire object with the id, make more sense to me (and I guess is the usual way) 🥇